src/org/usfirst/frc/team694/robot/commands/auton/routines/TripleCubeSwitchAutonCommand.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc.team694.robot.commands.auton.routines;
+
+import org.usfirst.frc.team694.robot.commands.LiftMoveToBottomCommand;
+import org.usfirst.frc.team694.robot.commands.LiftMoveToHeightCommand;
+import org.usfirst.frc.team694.robot.commands.QuisitorAcquireCommand;
+import org.usfirst.frc.team694.robot.commands.QuisitorDeacquireCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DrivetrainMoveInchesEncoderCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DrivetrainRotateAbsoluteDegreesPIDCommand;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class TripleCubeSwitchAutonCommand extends CommandGroup {
+
+    public TripleCubeSwitchAutonCommand() {
+        addSequential(new DoubleCubeSwitchAutonCommand());
+        addSequential(new DrivetrainMoveInchesEncoderCommand(48, -0.25));
+        addSequential(new LiftMoveToBottomCommand());
+        addSequential(new QuisitorAcquireCommand());
```

Comment:
Do this after turning, make it parallel, and add a timeout so that it doesn't acquire forever